### ISSUE TITLE
Is it possible to have a SpringStackView?

### ISSUE THREAD

**Issue by edias** [MengTo/Spring#157]
I was wondering me if it would be possible to have a SpringStackView?

I tried to extend SpringView but I would need to instantiate Spring and it seems it is private at the moment. 

Any idea?

**Comment by txaiwieser** [comment]
Yes, you can make a SpringStackView, but indeed you need a public initializer, thats why i made this PR: #177 

If you this should be added to the main repo you can make comments there to support the merge. Thanks

**Comment by txaiwieser** [comment]
I have updated the pull request to Swift 3.0 #177
